bin/templates/scripts/cordova/lib/run.js

```diff
@@ -87,15 +88,13 @@ module.exports.run = function (runOptions) {
                         var payloadFolder = path.join(buildOutputDir, 'Payload');
 
                         // delete the existing platform/ios/build/device/appname.app
-                        return spawn('rm', [ '-rf', appFile ], buildOutputDir)
-                            .then(function () {
-                                // move the platform/ios/build/device/Payload/appname.app to parent
-                                return spawn('mv', [ '-f', appFileInflated, buildOutputDir ], buildOutputDir);
-                            })
-                            .then(function () {
-                                // delete the platform/ios/build/device/Payload folder
-                                return spawn('rm', [ '-rf', payloadFolder ], buildOutputDir);
-                            });
+                        shell.rm('-rf', appFile);
```

Comment:
Just a quick comment that we have taken efforts to use `fs-extra` instead of `shelljs` in some other parts of the Cordova project. I would favor we do something similar in `cordova-ios` someday in the future (should not block Cordova 9 release).

Reply:
Good call. I was mostly going for consistency here, which should make it easier to properly replace any usages in the future.